sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -176,9 +174,8 @@ abstract class Scrollable extends StatefulComponent {
     return scrollVelocity.clamp(_kMinFlingVelocity, _kMaxFlingVelocity) / _kMillisecondsPerSecond;
   }
 
-  EventDisposition _handlePointerDown(_) {
+  void _handleDragStart() {
     _stopAnimations();
```

Comment:
This isn't quite right because the animations should stop as soon as your finger hits the screen, even if you don't end up dragging it anywhere.

Maybe skip the Scrollable changes in this patch and just focus on dismissable?